docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-keycloak-oidc.md

```diff
@@ -56,8 +58,18 @@ If you have an existing configuration using the SAML protocol and want to switch
     `Mapper Type` | `Group Membership`
     `Token Claim Name` | `full_group_path`
     `Full group path` | `ON`
+    `Add to ID token` | `ON`
+    `Add to access token` | `ON`
```

Comment:
Same as the above comment are these additional fields added in any of the related keycloak tickets? I ask because the issue linked does not have the fields listed.

Reply:
Same as the above comment, this is just to align with an option available in keycloak. Currently, you have to guess what is the correct option.